diff --git a/patterns.js b/patterns.js
index e56a3b785df29e726194f2b30e86bb19a78ef902..62eaf0a6719d15912d55a196435542e1c8cf5707 100644
--- a/patterns.js
+++ b/patterns.js
@@ -172,7 +172,7 @@ module.exports = options => {
         let idx = i - 2;
         while (idx > 0 && 'tbody_open' !== tokens[--idx].type);
 
-        const calc = tokens[idx].meta.colsnum >> 0;
+        const calc = tokens[idx].meta?.colsnum >> 0;
         if (calc < 2) { return; }
 
         const level = tokens[i].level + 2;
